src/coord_dsl/metamodels/fsm.tx

```diff
@@ -0,0 +1,34 @@
+FSM:
```

Comment:
I'd add a field for `name=ID` or sth similar so you can refer to it from other models